src/sites/twitch-twilight/modules/chat/input.jsx

```diff
@@ -324,10 +324,18 @@ export default class Input extends Module {
 		inst.tempInput = '';
 		inst.messageHistoryPos = -1;
 
+		let element = this.fine.getChildNode(inst);
```

Comment:
`overrideChatInput(inst)` is only called the first time a particular chat input is handled, and is not called when a chat input is re-rendered. Getting your DOM element here and caching it isn't a bad idea, but it can potentially break.

Reply:
What would be a better way? Like how do i need to change it? @SirStendec 

Sorry for the very late replay, was busy on other things but will now be accessable again